pyproject.toml

```diff
@@ -15,7 +15,7 @@ classifiers = ["Topic :: Software Development"]
 
 # Requirements
 dependencies = [
-    "vyper>=0.4.0",
+    "vyper>=0.4.0,<0.5.0",
```

Comment:
why add this change?

Reply:
Based on what I learned about pypi versioning, this seems like the right way without breaking stuff. But unrelated to this PR.